docs/introduction-to-smart-contracts.rst

```diff
@@ -304,7 +304,7 @@ These blocks form a linear sequence in time, and that is where the word "blockch
 Blocks are added to the chain at regular intervals, although these intervals may be subject to change in the future.
 For the most up-to-date information, it is recommended to monitor the network, for example, on `Etherscan <https://etherscan.io/chart/blocktime>`_.
 
-As part of the "order selection mechanism" (which is called "mining") it may happen that
+As part of the "order selection mechanism" (which is called "attestation") it may happen that
```

Comment:
If we're changing this, we should also deal with some other references to mining. For example [in the description of `coinbase()` opcode in Yul](https://docs.soliditylang.org/en/latest/yul.html#evm-dialect) and in the cheatsheet or in the description of the `receive` function. There's even one in the note below.